list

```diff
@@ -710,12 +726,15 @@ lihipro.com
 lihivip.com
 liip.to
 lin.ee
+lingolink.ru
```

Comment:
I'd need some working samples for it.

Reply:
The homepage is still Welcome to Nginx though.

Reply:
Seems to be entirely offline now. Will remove if it doesn't come back up.